CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## Supported Release 3.0.0
+### Summary
+
+This is a backwards incompatible major release.
+
+### Features
+
+- Docs updated for puppet-strings
+- Adds Puppet 4 data types and deprecation warnings
```

Comment:
This needs a documented section for the breaking change in addition to the list of added features.

IMHO according to http://keepachangelog.com/ we should have `### Added` for new features, and `### Changed` for the backwards-incompatible changes.

(And `### Fixed` for bugfixes, if there were any.)